### ISSUE TITLE
Split clip 30 min limit?

### ISSUE THREAD

**Issue by LarryM04** [OpenShot/openshot-qt#778]
**System Details:**
* Operating System / Distro: Win 7
* OpenShot Version: ? 2.3.4
* _Please attach log files if crash_

**Issue Description and steps to reproduce:**
I have a MP4 file that's an hour and 58 minutes long.  I would like to split it into smaller segments.  Per the Openshot doc, I dragged the file into the project file area, then right clicked and selected "Split Clip".  The first clip, about 26 minutes in length went fine, but I can't get the video to go past 30:00:04... the player just stops there.

In the end all I really want to do is chop this MP4 file into 4-5 smaller files.  Is there another way to do this?

**Comment by Just-Curious** [comment]
I suspected that maybe splitting and then splitting the splits might work.  I tried with an hour long video.  It did not go well.

<iframe width="560" height="315" src="https://www.youtube.com/embed/K8fj02IsOto?ecver=1" frameborder="0" allowfullscreen></iframe>

![20170923_splitclip](https://user-images.githubusercontent.com/16257232/30777989-67794f14-a096-11e7-8d65-65630f68abde.jpg)

After the clips are made, it would be nice to have easy access to their durations, start and stop times, etc.  Alternatively, eliminate the Split Clip dialog entirely and allow sections of the video to be saved to the project after cutting?
